src/views/workflow-history/workflow-history-event-details-recursive/workflow-history-event-details-recursive.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import isObjectLike from 'lodash/isObjectLike';
+
+import getHistoryEventFieldRenderConfig from '../workflow-history-event-details/helpers/get-history-event-field-render-config';
+import WorkflowHistoryEventDetailsBaseValue from '../workflow-history-event-details-base-value/workflow-history-event-details-base-value';
+
+import getDetailsFieldLabel from './helpers/get-details-field-label';
+import { styled } from './workflow-history-event-details-recursive.styles';
+import { type Props } from './workflow-history-event-details-recursive.types';
+
+export default function WorkflowHistoryEventDetailsRecursive({
+  details,
+  prefix = '',
+  decodedPageUrlParams,
+}: Props) {
+  return (
+    <>
+      {Object.entries(details).map(([key, value], index) => {
+        const path = (prefix ? prefix + '.' : '') + key;
+
+        const renderConfig = getHistoryEventFieldRenderConfig({
```

Comment:
Why redoing logic here, cant we have everything figured out from the helper method?
This would simplify debugging by having single source of truth. 

The old helper could have worked by allowing it to return a type of entry or entry group and the display would be just matter of rendering group or a row

Reply:
The main problem I encountered with that approach was that it was not very recursion-friendly. Let me give it another try.

Reply:
What if the group can have entries or group entries nested would that work ?